puffin/src/trace.rs

```diff
@@ -485,6 +494,255 @@ pub struct Trace<PT: ProtocolTypes> {
     pub prior_traces: Vec<Trace<PT>>,
 }
 
+/// Store the result of a trace execution for displaying or serializing
+#[derive(Serialize)]
+#[serde(bound = "PT: ProtocolTypes")]
+pub struct ExecutionResult<PT: ProtocolTypes> {
+    put: String,
+    success: bool,
+    error: Option<String>,
```

Comment:
Is `success` required here? Or is it equivalent to have `error = None`?

Reply:
It is equivalent, it is fixed